data/json/monsters.json

```diff
@@ -5963,6 +5963,37 @@
     "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "BLEED", "CBM_OP", "ACIDPROOF", "NO_BREATHE", "REVIVES", "BONES"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_zombie_armored",
+    "name": "decayed paladin",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"dkgray_red",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":25,
+    "aggression":100,
+    "morale":100,
+    "speed":70,
+    "melee_skill":5,
+    "melee_dice":2,
+    "melee_dice_sides":6,
+    "melee_cut":0,
+    "dodge":0,
+    "armor_bash":24,
+    "armor_cut":24,
+    "vision_day":30,
+    "vision_night":3,
+    "item_chance":80,
+    "death_drops": "mon_zombie_armored_death_drops",
+    "luminance":0,
+    "hp":120,
+    "death_function":["NORMAL"], 
+    "description":"Despite being encased in a nigh impenetrable suit of power armor, something still managed killed this soldier.",
```

Comment:
"Managed to kill"
And the armor stats don't suggest anything near "nigh impenetrable".